code_search/docker/ks/deploy.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Default namespace to deploy the ksonnet component to
+NAMESPACE="kubeflow"
+# Providing negative value to kubeflow wait would wait for a week
+TIMEOUT="-1s"
+
+usage()
+{
+    echo "usage: deploy.sh
+    [--working_dir The base GCS bucket ]
```

Comment:
Why is working_dir just a GCS bucket and not a full path?
Should there be an argument specifying where the index is created so that we don't override existing indexes?